src/main/java/org/opentripplanner/routing/graph/Graph.java

```diff
@@ -394,6 +394,21 @@ public void removeAlertPatch(Edge edge, AlertPatch alertPatch) {
         return new AlertPatch[0];
     }
 
+
+    /**
+     * Get {@link AlertPatch} {@link Set} for the graph.
+     * @return The {@link AlertPatch} set for the graph
+     */
+    public Set<AlertPatch> getAlertPatches() {
+        Set<AlertPatch> result = new HashSet<>();
```

Comment:
Small thing, but this method's name and signature could be made a little clearer. It's too similar to `AlertPatch[] getAlertPatches(Edge edge)`. You might want to change it to `getAllAlertPatches`. Also, the other `getAlertPatches`returns an array where this one returns a `Set`. Would it make sense to have them have same return types to make them more 'symmetric'?

Reply:
Fixed